app/properties.py

```diff
@@ -0,0 +1,58 @@
+#
+# properties.py
+#
+
+import json
+
+class Properties:
+
+    def __init__(self, json):
+        self.site_title = json["site_title"]
+        self.display_title = json["display_title"]
```

Comment:
What's the difference between `site_title` and `display_title`?

Reply:
`site_title` is what the browser shows and `display_title` is what is shown on the page. Most of the time I imagine they're the same, but perhaps someone wants `Ben's Blog` to appear on the tab, but `flippin bits` to be the actual blog name

Reply:
We don't need it - mostly a carry over from my site issues :)

Reply:
I dig it